rag_api/main.py

```diff
@@ -132,7 +146,8 @@ async def load_model(doc_path: str, doc_filter: str):
 async def answer(question: str):
     """Provides an LLM response based on query."""
     # https://towardsdatascience.com/building-a-rag-chain-using-langchain-expression-language-lcel-3688260cad05
-
+    global vector_store
```

Comment:
Similar to above, I don't think you need global keyword for just "read-only" activities.

Reply:
agree